modules/plugin/shapefile/src/main/java/org/geotools/data/shapefile/ShapefileDumper.java

```diff
@@ -326,13 +344,28 @@ private void writeToShapefile(
     }
 
     /**
-     * Allows subsclasses to perform extra actions against a shapefile that was completely written
+     * Allows subsclasses to perform extra actions against a shapefile that was completely written.
+     *
+     * <p>By default, this method calls {@link #shapefileDumped(SimpleFeatureType)}.
      *
      * @param fileName
      * @param remappedSchema
      */
     protected void shapefileDumped(String fileName, SimpleFeatureType remappedSchema)
             throws IOException {
+        shapefileDumped(remappedSchema);
+    }
+
+    /**
+     * Allows subsclasses to perform extra actions against a shapefile that was completely written.
+     *
+     * <p>This method will not be called if {@link #shapefileDumped(String, SimpleFeatureType)} is
+     * overwritten.
+     *
+     * @param fileName
```

Comment:
filename is not actually a parameter in this method. Once fixed just go ahead and merge.